staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto

```diff
@@ -1044,6 +1044,16 @@ message WindowsContainerResources {
     int64 memory_limit_in_bytes = 4;
     // Specifies the size of the rootfs / scratch space in bytes to be configured for this container. Default: 0 (not specified).
     int64 rootfs_size_in_bytes = 5;
+    // Specifies the set of CPUs to affinitize for this container.
+    repeated WindowsCpuGroupAffinity affinity_cpus = 6;
+}
+
+// WindowsCpuGroupAffinity specifies the CPU mask and group to affinitize
+message WindowsCpuGroupAffinity {
```

Comment:
I think this could be WindowsCpuAffinity and have the same meaning, but I'll defer to your expertise.

Reply:
Named it so to keep it in line with this struct https://learn.microsoft.com/en-us/windows-hardware/drivers/ddi/miniport/ns-miniport-_group_affinity . I'll add in a comment to point to this struct in the proto file - is that ok?

Reply:
wfm :)